ext/telemetry/errors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(tracker) {
+  return {
+    trackUncaught: () => {
+      window.onerror = (msg, url, line, col, err) => {
+        var data = {
+          msg: msg,
+          url: url,
+          line: line,
+          col: col,
+        };
+        tracker.trackEvent("uncaught error", { uncaughtError: data });
+      };
+    },
+    ignoreUncaught: () => {
+      window.onerror = () => {};
```

Comment:
We probably don't want to have all uncaught exception ignored after that, as Atom is relying on them to report issues on Atom or on the package that triggered the error. See my comment above, I think using the node event is a better solution.